src/OpenTelemetry.Exporter.Ocagent/Implementation/SpanDataExtentions.cs

```diff
@@ -36,7 +36,7 @@ internal static Span ToProtoSpan(this ISpanData spanData)
                 return new Span
                 {
                     Name = new TruncatableString { Value = spanData.Name },
-                    Kind = spanData.Kind == SpanKind.Client ? Span.Types.SpanKind.Client : Span.Types.SpanKind.Server,
+                    Kind = spanData.Kind == SpanKind.Client || spanData.Kind == SpanKind.Consumer ? Span.Types.SpanKind.Client : Span.Types.SpanKind.Server,
```

Comment:
proto definition will have all these types. Please add TODO comment about it. Or mention this line in the https://github.com/open-telemetry/opentelemetry-dotnet/issues/19

Reply:
Added TODO and updated issue #19 too.